src/Kunstmaan/TranslatorBundle/Controller/TranslatorCommandController.php

```diff
@@ -71,9 +71,12 @@ public function importForcedAction()
 
     /**
      * @Route("/export", name="KunstmaanTranslatorBundle_command_export")
+     * @Deprecated Using the "KunstmaanTranslatorBundle_command_export" route is deprecated since KunstmaanTranslatorBundle 5.4 and will be removed in KunstmaanTranslatorBundle 6.0. Use the default adminlist bundle export functionality instead.
```

Comment:
```suggestion
     * @deprecated Using the "KunstmaanTranslatorBundle_command_export" route is deprecated since KunstmaanTranslatorBundle 5.4 and will be removed in KunstmaanTranslatorBundle 6.0. Use the default adminlist bundle export functionality instead.
```

`@deprecated` should be all lowercase otherwise this error is triggered

```
[Semantical Error] The annotation "@Deprecated" in method Kunstmaan\TranslatorBundle\Controller\TranslatorCommandController::exportAction() was never imported. Did you maybe forget to add a "use" statement for this annotation?
```